util/docker/docker.go

```diff
@@ -0,0 +1,67 @@
+package docker
```

Comment:
Updated with versioned package for Docker client and changed the example to use the same client as a library.